aloha-core/src/main/scala/com/eharmony/aloha/models/multilabel/MultilabelModel.scala

```diff
@@ -0,0 +1,387 @@
+package com.eharmony.aloha.models.multilabel
```

Comment:
Why bother to have a multilabel package that only has one class?

Reply:
To give the `multilabel` package object somewhere to live.